src/scanpy/get/_aggregated.py

```diff
@@ -376,10 +373,13 @@ def aggregate_dask(
     mask: NDArray[np.bool_] | None = None,
     dof: int = 1,
 ) -> dict[AggType, DaskArray]:
-    if not isinstance(data._meta, CSRBase | np.ndarray):
+    if not isinstance(data._meta, CSBase | np.ndarray):
         msg = f"Got {type(data._meta)} meta in DaskArray but only csr_matrix/csr_array and ndarray are supported."
         raise ValueError(msg)
-    if data.chunksize[1] != data.shape[1]:
+    # i.e., if  data._meta is CSR/np.ndarray, this is 1 because we use row-chunking, but otherwise 0 and column chunking
+    unchunked_axis = int(isinstance(data._meta, CSRBase | np.ndarray))
+    chunked_axis = (unchunked_axis - 1) % 2
```

Comment:
no need to use a comment if you just use simpler code

```suggestion
    chunked_axis, unchunked_axis = (0, 1) if isinstance(data._meta, CSRBase | np.ndarray) else (1, 0)
```